ci-operator/step-registry/ocm/ci/fastforward/ocm-ci-fastforward-ref.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "ocm/ci/fastforward/ocm-ci-fastforward-ref.yaml",
+	"owners": {
```

Comment:
Don't forget our newest CICD member, MacdonEm - and we definitely don't want that TheRealHaoLiu guy in there.

Reply:
These were autogenerated from an OWNERS file higher up in the directory structure. If I change that file with this PR, it will update ALL of our Prow workflow steps and clutter up this PR. I'll do a separate PR that changes the OWNERS file.